pyproject.toml

```diff
@@ -53,13 +53,13 @@ httpx = {version = "^0.23.0", optional = true}
 isort = { version = "^5.5.3", optional = true}
 pdbpp = { version = "^0.10.2", optional = true}
 psycopg2 = { version = "^2.8.6", optional = true}
-pylama = {version = "^8.4.1", optional = true}
 pytest = {version = "^7.2.1", optional = true}
 pytest-asyncio = {version = "^0.19.0", optional = true}
 pytest-cov = { version = "^2.10.1", optional = true}
+ruff = {version = "^0.4.5", optional = true}
```

Comment:
Should we also remove black and isort?

Reply:
Oups, thanks [772a132](https://github.com/dialoguemd/fastapi-sqla/pull/131/commits/772a1327f8ad40d9e0b90bc36e28b54eb8736b77)